feature-utils/poly-import/src/entities/data-account.js

```diff
@@ -0,0 +1,57 @@
+export default class DataAccount {
+    constructor() {
+        this._importingResults = [];
+        this._importedFileNames = [];
+
+        this._processedData = [];
+        this._name = "";
+        this._preferredLanguage = [];
+        this._analysesExecutionResults = [];
+    }
+
+    get processedData() {
+        return this._processedData;
```

Comment:
In general, usages of this getter are:

```js
whatever.processedData[key] = value
```
or 

```js
whatever.processedData.key = value
```

Have you considered to turn it into

```
processedData( key, value)
````
?

That way, it would not violate encapsulation, allow us to add new business logic if needed, as well as refactor storage (using dependency injection, for instance) very easily.